.github/workflows/stable.yml

```diff
@@ -37,22 +37,19 @@ jobs:
   windows:
     runs-on: windows-latest
     permissions:
-      contents: write 
+      contents: write
     strategy:
       matrix:
+        version: [9.11.0, 9.12.0]
         include:
           - abi: cp38
             python-version: '3.8'
-            version: 9.11.0
```

Comment:
I think this is still needed (update it)

Reply:
I tkink it should work https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations but i haven't tested it.